### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by smk762**

### CODE REVIEW

**Review by tonymorony — COMMENTED** [review]
`Insall dependancies` - two typos I found there

File: README.md [hunk 1]
```diff
@@ -4,7 +4,10 @@
 
 Below is the list of prerequisites to compile `komodo-ecosysboard` on your machine:
 
-- Golang 1.12.6 minimum
+- Golang 1.12.6 minimum (see https://golang.org/doc/install)
+
+Insall dependancies
+`sudo apt-get install build-essential`
```
Comment (Milerius) [inline README.md:10]: This instruction is out of scope i think

Comment (smk762) [reply README.md:10]: Probably so, though uninitiated users on fresh systems may post issue or request this in support channels.

**Comment by smk762** [comment]
> `Insall dependancies` - two typos I found there

Fixed after sleep

File: README.md (hunk 1 above)
Comment (Milerius) [reply README.md:10]: All users are not necessarily on Ubuntu, I for example myself develop all the program on OSX. The information add in the readme is too intrusive to my taste. The project is in golang, the user must read the instructions of how to install golang on his machine. On the other hand, we can specify that the program uses CGO, in Golang's documentation it is written that CGO requires gcc and it is up to the user to do the necessary to obtain this dependency.

Comment (Milerius) [reply README.md:10]: What do you think @ca333 should we integrate this information directly in the README.md ?

**Comment by smk762** [comment]
hows about "OS specific build tools (e.g. build-essentials on Linux, XCode on OSX or MSVC on Win)." as used on https://github.com/KomodoPlatform/atomicDEX-API ?
